make/integration-tests.mk

```diff
@@ -11,7 +11,7 @@ test-bare-k8s-integration: clean-cov generate fmt vet ginkgo ## Requires only ba
 	mkdir -p $(PROJECT_PATH)/coverage/bare-k8s-integration
 #	Check `ginkgo help run` for command line options. For example to filtering tests.
 # Run in series as limit of 1 Kuadrant CR (procs=1)
-	OPERATOR_NAMESPACE=kuadrant-system $(GINKGO) \
+	OPERATOR_NAMESPACE=kuadrant-system WITH_EXTENSIONS=false $(GINKGO) \
```

Comment:
An alternative would be to build the extensions too, even if we don't have specific tests.. what do you reckon?

Reply:
or disabled by default, and in the deployments set the `WITH_EXTENSIONS` to `true`